include/openthread/platform/border_routing.h

```diff
@@ -65,6 +65,18 @@ extern "C" {
  */
 extern void otPlatBorderRoutingProcessIcmp6Ra(otInstance *aInstance, const uint8_t *aMessage, uint16_t aLength);
 
+/**
+ * Used to notify the platform whether they should request an prefix via DHCPv6 PD for Thread interface.
+ * When aShouldRequest is false, the platform must release the prefix lease.
+ *
+ * Requires `OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE`.
+ *
+ * @param[in] aInstance A pointer to an OpenThread instance.
```

Comment:
```suggestion
 * @param[in] aInstance  A pointer to an OpenThread instance.
```
2 spaces after the paramter name.